assets/src/edit-story/components/IconSwitch/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import LockClosedIcon from './lock_closed.svg';
+import LockOpenIcon from './lock_open.svg';
+
+const StyledButton = styled.button`
+  border: none;
+  background: transparent;
+  display: block;
+	padding: 0;
+	width: ${ ( { width } ) => width }px;
+	height: ${ ( { height } ) => height }px;
+  min-width: initial;
+  cursor: pointer;
+	color: ${ ( { theme } ) => theme.colors.fg.v3 };
+	&:focus, &:active, &:hover {
+    opacity: 1;
+    outline: none;
+	}
+	svg {
+		width: ${ ( { width } ) => width }px;
+		height: ${ ( { height } ) => height }px;
+  }
```

Comment:
This indentation is still all over the place. It makes it hard to review (and will look weird for others with different tab settings, even if it looks right to you). Please fix and make sure your code lints.

Reply:
I revise my configuration to use tabs. It will not happen again. Thank you!